src/uvw/stream.hpp

```diff
@@ -155,8 +155,8 @@ class StreamHandle: public Handle<T, U> {
 
 public:
 #ifdef _WIN32
```

Comment:
This can be changed to `#ifdef _MSC_VER` instead. Or if you know which version of MSVC supports the inherited constructor form, `#if defined(_MSC_VER) && _MSC_VER < supported_version`.
The block here was added for not-fully-compliant compilers - primarily MSVC. GCC should be fine with the `#else` block.